packages/preview/vercanard/1.0.0/typst.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "vercanard"
+version = "1.0.0"
+entrypoint = "template.typ"
+authors = [ "Ana Gelez <https://ana.gelez.xyz>" ]
+license = "GPL-3.0"
+description = "A colorful CV template"
+repository = "https://github.com/elegaanz/vercanard"
+categories = [ "cv" ]
+keywords = [ "CV", "Resume", "Color" ]
+exclude = [ "*.pdf" ]
+
+[template]
+path = "."
+entrypoint = "template.typ"
```

```suggestion
path = "template"
entrypoint = "main.typ"
```
and then here the path should designate a directory that is copied to the user's project and the entrypoint is the file within it that is used as the compilation target.